### PR TITLE
Improve deprecation assignment with deprecated_argument_lookup

### PR DESCRIPTION
**Pull request by imsheridan**
This PR is to improve deprecation assignment with deprecated_argument_lookup, which is used in all other arguments deprecations.
The `tf.losses.cosine_distance` deprecated `dim` and switched to `axis`.

### CODE REVIEW

**Review by martinwicke — APPROVED** [review]

**Comment by martinwicke** [comment]
This is an odd failure, with no detail link. I'm assuming this is a Kokoro failure?
